index.html

```diff
@@ -22,6 +22,9 @@
 </head>
 
 <body>
+  <h1>私は1.8行に一回RequestChangesを受けました</h1>
```

Comment:
let's add more genkainess
```suggestion
  <h1>私は<b><i>1.8行に一回RequestChanges</i></b>を受けました</h1>
```

Reply:
HTML 5 だと `<strong>` と `<em>` を使うべきでは

Reply:
ミ゛